projects/admin/src/app/record/detail-view/document-detail-view/document-detail-view.component.html

```diff
@@ -518,8 +512,20 @@ <h3 id="{{'doc-altgr-title-' + i}}">{{ altgr_title }}</h3>
         </div>
       </tab>
       <!-- END OF DESCRIPTION TAB --------------------------------------------->
+
+      <!-- LOCAL FIELDS TAB ------------------------------------------------------------>
+      <tab id="documents-local-field-tab" tabOrder="4" *ngIf="!record.metadata.harvested">
+        <ng-template tabHeading>
+          <i class="fa fa-list-ul mr-1"></i><span translate>Local fields</span>
```

Comment:
{{ 'Local fields' | translate }} instead of `<span transalte>`

Reply:
done